sotodlib/preprocess/processes.py

```diff
@@ -1355,6 +1357,28 @@ def save(self, proc_aman, split_flg_aman):
         if self.save_cfgs:
             proc_aman.wrap("split_flags", split_flg_aman)
 
+class UnionFlags(_Preprocess):
+    """Do the union of relevant flags for mapping
+    Typically you would include turnarounds, glitches, etc.
+
+    Saves results for aman under the "flags.glitch_flags" field.
+
+     Example config block::
+
+        - name : "union_flags"
+          process:
+            flag_labels: ['jumps_2pi.jump_flag', 'glitches.glitch_flags', 'turnaround_flags.turnarounds']
```

Comment:
Possibly unrelated to this PR specifically, but I do see that `aman.flags` is already populated with `left_scan[samps]`, `right_scan[samps]`, and `turnarounds[samps]` (`merge=True` by default) but not by `glitch_flags` or `jump_flags` (`merge=False` is hardcoded).  These should probably standardized in another PR.  If all flags are merged into `aman.flags` by default, we could just search and merge everything in `aman.flags` instead of specifying them via the config file.

Reply:
So by default, an obs has an empty `FlagManager` in `aman.flags`. When you do the turnarounds function the `left_scan`, `right_scan`, and `turnarounds` flags are populated. I think we still should keep some amount of control such that in the preprocess config we can specify exactly what we want to go into the overall glitch flags.

Reply:
I agree that we want to keep the flexibility in which flags get combined to form the set of flags used in the mapmaking and not blinding search for all flags and combine them.